### ISSUE TITLE
Route::crud - allow passing a class instead of string

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/community-forum#231]
@promatik mentioned it would be cool for us to be able to do:
```php
Route::crud('products', ProductCrudController::class);
```

not only

```php
Route::crud('products', 'ProductCrudController');
```

**Comment by tabacitu** [comment]
@karandatwani92 this is a cool little feature - you want to do it?

**Comment by tabacitu** [comment]
> Would it become Backpack\CRUD\app\Http\Controllers\App\Other\MyController? I don't think we can ignore the namespace configuration in the route group.

Damn. Didn't think of that. Good catch, Pedro. How does Laravel do it for its own routes? 

What I find intuitive... if you use `::class`, that should be the full path. So it basically ignores the `namespace`. You either specify the full path, or import the class... which IDEs make easy. Not 100% sure though...

**Comment by karandatwani92** [comment]
> @karandatwani92 this is a cool little feature - you want to do it?

Yes, I'll do it. Watching yours and Pedro's discussion on it, Once we reach a direction. I'll make the changes.

**Comment by tabacitu** [comment]
Turns out we had a discussion about this too - https://github.com/Laravel-Backpack/community-forum/discussions/332 🤦‍♂️

**Comment by karandatwani92** [comment]
I tested this on the latest version, and …

(1) support for this syntax is already working fine.
(2) But when I made the following default one inside(AddCustomRouteContent.php):
/App/Http/Controllers/Admin/{nameTitle}CrudController::class)

One has to be manually remove namespace attribute from Route:group and append namespace to other routes for once.
Then this would become compatible for future usage.

@tabacitu What should we do?
